libraries/AC_PrecLand/AC_PrecLand_IRLock.h

```diff
@@ -36,11 +36,15 @@ class AC_PrecLand_IRLock : public AC_PrecLand_Backend
 
     // return true if there is a valid los measurement available
     bool have_los_meas() override;
+    
+    // return target type
+    LANDING_TARGET_TYPE target_type() override;
 
 private:
     AP_IRLock_I2C irlock;
 
     Vector3f            _los_meas_body;         // unit vector in body frame pointing towards target
     bool                _have_los_meas;         // true if there is a valid measurement from the camera
     uint32_t            _los_meas_time_ms;      // system time in milliseconds when los was measured
+    LANDING_TARGET_TYPE _target_type;           // LANDING_TARGET_TYPE enum
```

Comment:
Unused.

Reply:
Connected frontend target_type() to backend target_type()